outreach/write.go

```diff
@@ -0,0 +1,13 @@
+package outreach
```

Comment:
Same as the metadata function, can be removed because of https://github.com/amp-labs/connectors/pull/195